system_tests_tcp_adaptor times out on test_12_tcp_EA2_EC1_5000000

Sometimes, the system_tests_tcp adaptor times out when executing the test - test_12_tcp_EA2_EC1_5000000

```
 59: DEBUG    root:system_test.py:1696 2022-10-03 19:08:19.760933 TcpAdaptor: ECHO_SERVER TcpAdaptor ES_EC1 write to : 127.0.0.1:46594 len:1024: b'aaaaaaaaaaaa[localhost:24895:0:4285450]aaaaaaaaaaa' ... b'4895:0:4286400]aaaaaaaaaaaaaaaaaaaaaaa[localhost:2'
59: DEBUG    root:system_test.py:1696 2022-10-03 19:08:19.761082 TcpAdaptor: ECHO_SERVER TcpAdaptor ES_EC1 read from: 127.0.0.1:46594 len:1024: b'4895:0:4286450]aaaaaaaaaaaaaaaaaaaaaaa[localhost:2' ... b'aaaaaaaaaaaaaa[localhost:24895:0:4287450]aaaaaaaaa'
59: DEBUG    root:system_test.py:1696 2022-10-03 19:08:19.761223 TcpAdaptor: ECHO_SERVER TcpAdaptor ES_EC1 write to : 127.0.0.1:46594 len:1024: b'4895:0:4286450]aaaaaaaaaaaaaaaaaaaaaaa[localhost:2' ... b'aaaaaaaaaaaaaa[localhost:24895:0:4287450]aaaaaaaaa'
59: DEBUG    root:system_test.py:1696 2022-10-03 19:08:19.803188 TcpAdaptor-testClass: TCP_TEST test_12_tcp_EA2_EC1_5000000 failed: TCP_TEST TIMEOUT - local wait time exceeded
59: DEBUG    root:system_test.py:1696 2022-10-03 19:08:19.761378 TcpAdaptor: ECHO_SERVER TcpAdaptor ES_EC1 read from: 127.0.0.1:46594 len:1024: b'aaaaaaaaaaaaaa[localhost:24895:0:4287500]aaaaaaaaa' ... b':24895:0:4288450]aaaaaaaaaaaaaaaaaaaaaaa[localhost'
59: ------------ generated xml file: /home/runner/work/skupper-router/skupper-router/skupper-router/build/tests/junitxmls/system_tests_tcp_adaptor.xml -------------
59: =================================================================== short test summary info ====================================================================
59: FAILED ::TcpAdaptor::test_14_tcp_EA2_EC1_5000000 - AssertionError: TCP_TEST Stop test_12_tcp_EA2_EC1_5000000 FAIL: TCP_TEST TIMEOUT - local wait time exceeded
59: ===================================================== 1 failed, 18 passed, 1 skipped in 445.81s (0:07:25) ======================================================
```

https://github.com/skupperproject/skupper-router/actions/runs/3176719775/jobs/5176373460#step:10:13428

--- a/tests/system_tests_tcp_adaptor.py
+++ b/tests/system_tests_tcp_adaptor.py
@@ -859,12 +859,12 @@ class TcpAdaptor(TestCase):
         assert result is None, "TCP_TEST Stop %s FAIL: %s" % (name, result)
 
     @unittest.skipIf(DISABLE_SELECTOR_TESTS, DISABLE_SELECTOR_REASON)
-    # This test sends and receives 5 million bytes from edge router to edge router
+    # This test sends and receives 2 million bytes from edge router to edge router
     # which will make the code go in and out of the TCP window.
-    def test_14_tcp_EA2_EC1_5000000(self):
-        name = "test_12_tcp_EA2_EC1_5000000"
+    def test_14_tcp_EA2_EC1_2000000(self):
+        name = "test_12_tcp_EA2_EC1_2000000"
         self.logger.log("TCP_TEST Start %s" % name)
-        pairs = [self.EchoPair(self.EA2, self.EC1, sizes=[5000000])]
+        pairs = [self.EchoPair(self.EA2, self.EC1, sizes=[2000000])]
         result = self.do_tcp_echo_n_routers(name, pairs)
         if result is not None:
             print(result)

